guides/standards-assurance-tables.md

```diff
@@ -0,0 +1,158 @@
+---
+expires: 2019-09-01
+---
+# Standards Assurance Tables
+
+The MOJ Cyber Security team have developed a 'Standards Assurance Table' (SAT) in the form of a Google Sheet template.
+
+The SAT measures technology systems (and surrounding information governance) against the [UK Cabinet Office](https://www.gov.uk/government/organisations/cabinet-office) [Minimum Cyber Security Standard (MCSS)](https://www.gov.uk/government/publications/the-minimum-cyber-security-standard) and [UK National Cyber Security Centre (NCSC)](https://www.ncsc.gov.uk/) [Cloud Security Principles (CSPs)](https://www.ncsc.gov.uk/collection/cloud-security?curPage=/collection/cloud-security/implementing-the-cloud-security-principles).
+
+For transparency and open-working purposes, a [redacted copy of the Standards Assurance Table](https://docs.google.com/spreadsheets/d/1oO9RAZQ6ETsN2iHZb5zcFM0HROliw18MBv89oF-BPeg/edit?usp=sharing) has been published. Please note, this is not the functional template used within the MOJ.
+
+## SAT Template
+
+The SAT itself is written to be self-explanitory to a cyber security professional that is already aware of the MCSS/CSP and has a familiarity with information risk management concepts.
+
+- Black labelled sheets describe the SAT and how it should be used
+- Blue labelled sheets are the sheets to be completed
+- Yellow labelled sheets are automatically calculated reporting based on the blue labelled sheet data
+- Green labelled sheets offer help/guidance on SAT components
+
+## Key SAT concepts
+
+The SATs have concepts of "Objectives", "Evidence", "Confidence", an overall "Delta" (which is the most pertinent output) and "Further Evidence Required", with supporting commentary.
+
+The primary SAT purpose is to assess the system against the MCSS/CSP, determining how compliant (or non-compliant) the system is, based on a collation and review of evidence. Evidence is analysed to determine confidence that the evidence demonstrates the system meets (or does not meet) the standard, and the 'gap' (delta) between system posture according to said evidence and the standards.
+
+### Objectives
+
+The MCSS/CSPs have been distilled into 39 Objectives which the assessor (typically a cyber security professional) completing the SAT is measuring against.
```

Comment:
Objectives which the assessor (typically a cyber security professional) completing the SAT is measuring against. -> The Assessor (normally a cyber security professional) completes the SAT by evaluating the target system against the objectives.